CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - Add reason codes to the `X-Sentry-Rate-Limits` header in store responses. This allows external Relays to emit outcomes with the proper reason codes. ([#850](https://github.com/getsentry/relay/pull/850))
 - Emit metrics for outcomes in external relays. ([#851](https://github.com/getsentry/relay/pull/851))
 - Make `$error.value` `pii=true`. ([#837](https://github.com/getsentry/relay/pull/837))
+- Send network outage metric on healthcheck endpoint hit. ([#856](https://github.com/getsentry/relay/pull/856))
```

Comment:
This could even be moved under "Features".

Reply:
Moved to the features section.